docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:bionic
+
+RUN apt update
+RUN apt -y install \
+		sudo \
+		# openssh-server \
+		git \
+		npm
+
+RUN useradd kaya --uid 1000 -m
+
+EXPOSE 4200
+
+WORKDIR /home/kaya
+USER kaya
+
+RUN git clone https://github.com/yoeriapts/kaya.git && \
+    cd kaya && \
+    npm install
+
+ENTRYPOINT cd kaya && \
```

Comment:
It may be better to use `node` as the `ENTRYPOINT` for this particular image, since I foresee that it will be used as a binary more than as a "VM", so to say.